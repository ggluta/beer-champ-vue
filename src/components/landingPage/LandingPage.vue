<template>
  <v-container id="landing-page" class="pa-0" fluid>
    <v-img src="../../assets/logo.png"/>
    <div class="my-5">
      <DropdownMenu></DropdownMenu>
      <Gallery></Gallery>
    </div>
  </v-container>
</template>

<script>
import gsap from 'gsap';
import Gallery from '../gallery/Gallery.vue';
import DropdownMenu from '../dropdownMenu/DropdownMenu.vue';

export default {
  name: 'LandingPage',
  components: { DropdownMenu, Gallery },
  mounted() {
    gsap.fromTo('#landing-page', { yPercent: 10, autoAlpha: 0 }, {
      yPercent: 0, autoAlpha: 1, duration: 4.5, ease: 'power4.out',
    });
  },
};
</script>
