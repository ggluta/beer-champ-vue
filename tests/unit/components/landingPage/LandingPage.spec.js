import { shallowMount } from '@vue/test-utils';
import LandingPage from '../../../../src/components/landingPage/LandingPage.vue';

describe('Gallery test', () => {
  test('SHOULD render default WHEN normally instantiated', () => {
    const wrapper = shallowMount(LandingPage);

    expect(wrapper).toMatchSnapshot();
  });
});
