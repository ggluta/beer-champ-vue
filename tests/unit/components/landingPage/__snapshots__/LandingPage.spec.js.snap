// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery test SHOULD render default WHEN normally instantiated 1`] = `
<v-container id="landing-page" fluid="" class="pa-0">
  <v-img src="../../assets/logo.png"></v-img>
  <div class="my-5">
    <dropdownmenu-stub></dropdownmenu-stub>
    <gallery-stub></gallery-stub>
  </div>
</v-container>
`;
